### ISSUE TITLE
Allow to configure annotations on Front-LoadBalancers

### ISSUE THREAD

**Issue by xrstf** [kubermatic/kubermatic#9301]
### What would you like to be added? / User Story
For exposeStrategy=NodePortProxy we already support setting `seed.spec.nodePortProxy.annotations` to allow admins to tweak the LB service. For exposeStrategy=LoadBalancer, there is currently no option to customize the Front-LoadBalancer (pkg/resources/nodeportproxy/nodeportproxy.go)

One approach woul be to simply re-use the annotations from the NodePortProxy, since no seed can have both strategies at the same time. On the other hand, this could be also be a surprising.

### Acceptance criteria
* There is a way to configure annotations on the front LB.

**Comment by ahmedwaleedmalik** [comment]
@xrstf Since these services are mutually exclusive as we can't have both expose strategies within the same seed. I'd say it's fine to re-use these annotations. WDYT? I'm working on something similar for another ticket so I can take care of this alongside as well.

Also, should this really be an epic?

**Comment by xrstf** [comment]
I didn't make this an epic and the only downside of re-using those annotations is that they are called "nodePortProxy" in the Seed, so it might get confusing. But I'm not totally against it.

**Comment by ahmedwaleedmalik** [comment]
How about:

```yaml
spec:
  nodeportProxy:
    annotations:
      key: value
  envoy:
    annotations:
      key: value
    loadBalancerSourceRanges:
    - "143.231.0.0/16"
```

`loadBalancerSourceRanges` is something I'm planning to add against https://github.com/kubermatic/kubermatic/issues/7048

**Comment by tboerger** [comment]
How does the shifting of the annotations and the source range addition resolve the initial issue?

**Comment by ahmedwaleedmalik** [comment]
@tboerger you can set the annotations for the front-loadbalancer using that annotation now.

https://github.com/kubermatic/kubermatic/blob/master/pkg/resources/nodeportproxy/nodeportproxy.go#L405

**Comment by tboerger** [comment]
Ok, so I will be able to do that starting with 2.21, before that I'm forced to use NodePort and use larger loadbalancers for Hetzner Cloud.

**Comment by ahmedwaleedmalik** [comment]
Yes, this will also be shipped with the next patch release for 2.20 i.e. 2.20.2
